src/main/java/bio/terra/cli/apps/DockerAppsRunner.java

```diff
@@ -38,8 +39,11 @@
   private final WorkspaceContext workspaceContext;
   private DockerClient dockerClient;
 
-  // This is where the pet key files will be mounted on the Docker container.
-  public static final String PET_KEYS_MOUNT_POINT = "/usr/local/etc/terra_cli";
+  // default $HOME directory on the container (this is where we expect to look for the global
+  // context)
+  private static final String CONTAINER_HOME_DIR = "/root";
+  // mount point for the workspace directory
```

Comment:
I don't have strong opinions about comment style, so that change sounds fine to me. However, I prefer consistency within the codebase, and changing this here would make it inconsistent with how I've done comments throughout. Perhaps add a new ticket with any style changes you'd like to see and we can take care of them all at once in a separate PR.